db/migrate/20170501201633_add_canceled_to_order_detail.rb

```diff
@@ -0,0 +1,19 @@
+class AddCanceledToOrderDetail < ActiveRecord::Migration
+  def up
+    add_column :order_details, :canceled_at, :datetime
+    add_column :order_details, :canceled_by, :integer
+    add_column :order_details, :canceled_reason, :string
+
+    Reservation.find_each do |reservation|
+      reservation.order_detail.update_attributes(canceled_at: reservation.canceled_at,
+                                                 canceled_by: reservation.canceled_by,
+                                                 canceled_reason: reservation.canceled_reason)
```

Comment:
I didn't add the remove_columns into this migration because I thought perhaps we would prefer to try this on stage first and verify that the data migration went smoothly before dropping a bunch of data.  I'm not sure what the best strategy is for data migrations like this--might be good for the two of us to discuss.